### PR TITLE
Added my project labforum

### PR DESCRIPTION
**Pull request by littlephone**

### CODE REVIEW

File: _data/projects/Labforum.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://www.labstry.com
+tags:
+  - php
+  - web
+  - html
+  - cross-platforum
+  - forum
+upforgrabs:
+  name: help wanted
+  link: https://github.com/labstry-opensource/labforum
```
Comment (ritwik12) [inline _data/projects/Labforum.yml]: @littlephone Please change this to https://github.com/labstry-opensource/labforum/labels/help%20wanted

**Review by ritwik12 — DISMISSED** [review]

File: _data/projects/Labforum.yml (hunk 1 above)
Comment (littlephone) [reply _data/projects/Labforum.yml]: @ritwik12 The change is completed.

**Comment by ritwik12** [comment]
@littlephone Please check again I am not sure it is. I am not able to see that here.

**Review by shiftkey — APPROVED** [review]

**Comment by ritwik12** [comment]
Thanks @littlephone and @shiftkey

**Pull request merged**